lib/commands/winapi/user32.js

```diff
@@ -205,6 +213,15 @@ export async function handleInputs(inputs) {
   return uSent;
 }
 
+export const ensureDpiAwareness = _.memoize(async function ensureDpiAwareness() {
+  await getUser32().SetProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2);
```

Comment:
Could this method leave a log when `SetProcessDpiAwarenessContext` method gets false to leave the set failure as a hint? 
https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setprocessdpiawarenesscontext

Reply:
Added logging